src/presentation/webapp/pages/store-list/StoreListPage.tsx

```diff
@@ -0,0 +1,210 @@
+    }, [getStores, objectsTableKey]);
+
+    const backHome = () => history.push("/");
+
+    const handleCreateStore = () => history.push(`/stores/new`);
+
+    const handleEditStore = (id: string) => {
+        history.push(`/stores/edit/${id}`);
+    };
+
+    const handleSetStoreAsDefault = async (id: string) => {
+        const result = await setStoreAsDefault(id);
+        result.match({
+            error: () => {
+                snackbar.error(i18n.t("An error has occurred setting store as default"));
+            },
+            success: () => setObjectsTableKey(Math.random()),
+        });
+    };
+
+    const validateError = (error?: GitHubError): string => {
+        switch (error) {
+            case "NO_TOKEN":
+                return i18n.t("The token is empty");
+            case "NO_ACCOUNT":
+                return i18n.t("The account is empty");
+            case "NO_REPOSITORY":
+                return i18n.t("The repository is empty");
+            case "BAD_CREDENTIALS":
+                return i18n.t("The token is invalid");
+            case "NOT_FOUND":
+                return i18n.t("Repository not found");
+            case "UNKNOWN":
+            default:
+                return i18n.t("Unknown error");
+        }
+    };
+
+    const handleTestConnection = async (id: string) => {
+        loading.show(true, i18n.t("Testing GitHub connection"));
+
+        const store = await compositionRoot.store.get(id);
+
+        if (store) {
+            const validation = await compositionRoot.store.validate(store);
+            validation.match({
+                error: error => {
+                    snackbar.error(validateError(error));
+                },
+                success: () => {
+                    snackbar.success(i18n.t("Connected successfully"));
+                },
+            });
+        } else {
+            snackbar.error(i18n.t("Store not found:" + id));
+        }
+
+        loading.reset();
+    };
+
+    const updateTable = ({ selection }: TableState<Store>) => {
+        setSelection(selection);
+    };
+
+    const confirmDelete = async () => {
+        loading.show(true, "Deleting stores");
+
+        for (const id of toDelete) {
+            await deleteStore(id);
+        }
+
+        setObjectsTableKey(Math.random());
+        setSelection([]);
+        setToDelete([]);
+        loading.reset();
+    };
+
+    const columns: TableColumn<Store>[] = [
+        { name: "id", text: i18n.t("Id"), sortable: true, hidden: true },
+        { name: "account", text: i18n.t("Account"), sortable: true },
+        { name: "repository", text: i18n.t("Repository"), sortable: true },
+        { name: "token", text: i18n.t("Token"), sortable: true, hidden: true },
+        {
+            name: "default",
+            text: i18n.t("Default"),
+            sortable: true,
+            getValue: store => {
+                return <Checkbox disabled={true} checked={store.default} />;
```

Comment:
![image](https://user-images.githubusercontent.com/2181866/97298619-fd8a8d00-1853-11eb-8c9b-c4563e51c284.png)

```suggestion
                return store.default ? <Icon>done</Icon> : "";
```

Reply:
thanks @SferaDev